shared/src/hooks/useLatestAPY.ts

```diff
@@ -107,6 +113,7 @@ export const calculateAPYFromPriceHistory = (
   }
 
   let weeksAgo = 0;
+  const apys: number[] = [];
```

Comment:
Here we create an array to keep track of the APY calculation between each rounds